libs/wire-subsystems/test/unit/Wire/MiniBackend.hs

```diff
@@ -1,3 +1,5 @@
+{-# OPTIONS_GHC -Wno-ambiguous-fields #-}
+
```

Comment:
do we need this one? there doesn't seem to be any change here. 
```suggestion
```

Reply:
Yeah, it's needed because `status` is now abiguous.